### ISSUE TITLE
Data file paths need updating

### ISSUE THREAD

**Issue by danjgale** [miykael/mni_atlas_reader#10]
While I was working on documentation and testing out some functionalities, I had to fix the data file paths because the `atlas` and `templates` directories have been relocated. My solution – only to work in a pinch – was just adding relative paths in the existing code. A longer term and a more stable solution is needed.

**Comment by rmarkello** [comment]
This is a direct consequence of my moving too quickly on #1, where I moved `atlas_reader.py` into a sub-folder and left `atlases` and `templates` in the root directory of the repository. My bad!!

One solution to this might be to bundle the `atlases` and `templates` within the package via the `package_data` keyword argument in our [`setup.py` script](https://github.com/miykael/mni_atlas_reader/blob/aca438d/setup.py). 

To do this, we can create a new folder in `mni_atlas_reader` (called, for example, `data`) and put the two data folders inside there! The layout would thus look something like (ignoring all the non-relevant files):

```
└── mni_atlas_reader/
    ├── __init__.py
    ├── atlas_reader.py
    ├── info.py
    └── data/ 
        ├── templates/ 
        └── atlases/ 
```

We could then ensure these data files are packaged with the project by editing `PACKAGE_DATA` in [`info.py`](https://github.com/miykael/mni_atlas_reader/blob/aca438d/mni_atlas_reader/info.py#L28) to look like:

```python
PACKAGE_DATA = {
    'mni_atlas_reader': ['data/*']
}
```

That will install all the atlases and templates with `mni_atlas_reader`! Then we can use [`pkg_resources`](https://setuptools.readthedocs.io/en/latest/pkg_resources.html) to dynamically access them within the main functionality of the project.

Assuming we did that, the basic function call for accessing data (say, the `atlas_aal.mgz` file) stored within the project would be:

```python
import pkg_resources
atlas_path = pkg_resources.resource_filename('mni_atlas_reader', 'data/atlases/atlas_aal.mgz')
```

In this example, `atlas_path` would then be the absolute path (on the local machine) to that file and could be loaded as such. I vote we implement this right now to ensure the package functions and then revisit this whole "bundling" issue when progress is made on #4!
